tutoraspects/plugin.py

```diff
@@ -366,7 +366,7 @@
         # For now we are pulling this from github, which should allow maximum
         # flexibility for forking, running branches, specific versions, etc.
         ("DBT_REPOSITORY", "https://github.com/openedx/aspects-dbt"),
-        ("DBT_BRANCH", "v3.29.1"),
+        ("DBT_BRANCH", "bmtcril/remove_unused_models"),
```

Comment:
don't forget to change this after you push https://github.com/openedx/aspects-dbt/pull/111